atomic_reactor/plugins/pre_fetch_sources.py

```diff
@@ -238,6 +242,16 @@ def get_srpm_urls(self, sigkeys=None, insecure=False):
             if 'SOURCERPM' not in rpm_hdr:
                 raise RuntimeError('Missing SOURCERPM header (RPM ID: {})'.format(rpm_id))
 
+            skip_srpm = False
+            srpm_name = rpm_hdr['SOURCERPM'].split('-', 2)[0]
```

Comment:
Was the `split` supposed to be `rsplit`?

Reply:
Nice catch!

Reply:
yeah it was, good catch